app/models/govuk_publishing_components/audit_comparer.rb

```diff
@@ -18,51 +20,58 @@ def sort_results(results)
       data = []
 
       results.each do |result|
-        templates = result[:components_found].find { |c| c[:location] == "templates" }
-        stylesheets = result[:components_found].find { |c| c[:location] == "stylesheets" }
-        print_stylesheets = result[:components_found].find { |c| c[:location] == "print_stylesheets" }
-        javascripts = result[:components_found].find { |c| c[:location] == "javascripts" }
-        ruby = result[:components_found].find { |c| c[:location] == "ruby" }
-
-        @all_stylesheets = true if stylesheets[:components].include?("all")
-        @all_print_stylesheets = true if print_stylesheets[:components].include?("all")
-        @all_javascripts = true if javascripts[:components].include?("all")
-
-        templates[:components] = include_any_components_within_components(templates[:components])
-
-        warnings = []
-        warnings << warn_about_missing_components(result[:components_found])
-        warnings << warn_about_missing_assets(result[:components_found])
-        warnings = warnings.flatten
-
-        data << {
-          name: result[:name],
-          application_found: result[:application_found],
-          summary: [
-            {
-              name: "Components in templates",
-              value: templates[:components].flatten.uniq.sort.join(", "),
-            },
-            {
-              name: "Components in stylesheets",
-              value: stylesheets[:components].join(", "),
-            },
-            {
-              name: "Components in print stylesheets",
-              value: print_stylesheets[:components].join(", "),
-            },
-            {
-              name: "Components in javascripts",
-              value: javascripts[:components].join(", "),
-            },
-            {
-              name: "Components in ruby",
-              value: ruby[:components].join(", "),
-            },
-          ],
-          warnings: warnings,
-          warning_count: warnings.length,
-        }
+        if !result[:application_found]
```

Comment:
Does this mean the same thing using `unless`? For example: 
```ruby
unless result[:application_found]
```

Reply:
Yes, changed, thanks.